src/main/java/ch/njol/skript/registrations/EventValues.java

```diff
@@ -241,163 +244,198 @@ public static <T, E extends Event> Kleenean hasMultipleConverters(Class<E> event
 	@Nullable
 	@Deprecated(forRemoval = true)
 	@SuppressWarnings({"removal"})
-	public static <T, E extends Event> Getter<? extends T, ? super E> getEventValueGetter(Class<E> event, Class<T> type, int time) {
-		return toGetter(getEventValueConverter(event, type, time, true));
+	public static <T, E extends Event> Getter<? extends T, ? super E> getEventValueGetter(Class<E> eventClass, Class<T> valueClass, int time) {
+		return toGetter(getEventValueConverter(eventClass, valueClass, time, true));
 	}
 
 	/**
-	 * Returns a {@link Converter} to get a value from in an event.
+	 * Returns a {@link Converter} to get a value from in an eventClass.
 	 * <p>
-	 * Can print an error if the event value is blocked for the given event.
+	 * Can print an error if the eventClass value is blocked for the given eventClass.
 	 *
-	 * @param event the event class the {@link Converter} will be getting from.
-	 * @param type type of {@link Converter}.
-	 * @param time the event-value's time.
+	 * @param eventClass the eventClass class the {@link Converter} will be getting from.
+	 * @param valueClass type of {@link Converter}.
+	 * @param time the eventClass-value's time.
 	 * @return A getter to get values for a given type of events.
 	 * @see #registerEventValue(Class, Class, Converter, int)
 	 * @see EventValueExpression#EventValueExpression(Class)
 	 */
 	public static <T, E extends Event> @Nullable Converter<? super E, ? extends T> getEventValueConverter(
-		Class<E> event, Class<T> type, int time
+		Class<E> eventClass, Class<T> valueClass, int time
 	) {
-		return getEventValueConverter(event, type, time, true);
+		return getEventValueConverter(eventClass, valueClass, time, true);
 	}
 
 	@Nullable
 	private static <T, E extends Event> Converter<? super E, ? extends T> getEventValueConverter(
-		Class<E> event, Class<T> type, int time, boolean allowDefault
+		Class<E> eventClass, Class<T> valueClass, int time, boolean allowDefault
 	) {
-		List<Converter<? super E, ? extends T>> list = getEventValueConverters(event, type, time, allowDefault);
+		List<Converter<? super E, ? extends T>> list = getEventValueConverters(eventClass, valueClass, time, allowDefault);
 		if (list == null || list.isEmpty())
 			return null;
 		return list.get(0);
 	}
 
 	@Nullable
 	private static <T, E extends Event> List<Converter<? super E, ? extends T>> getEventValueConverters(
-		Class<E> event, Class<T> type, int time, boolean allowDefault
+		Class<E> eventClass, Class<T> valueClass, int time, boolean allowDefault
 	) {
-		return getEventValueConverters(event, type, time, allowDefault, true);
+		return getEventValueConverters(eventClass, valueClass, time, allowDefault, true);
 	}
 
 	/*
-	 * We need to be able to collect all possible event-values to a list for determining problematic collisions.
+	 * We need to be able to collect all possible eventClass-values to a list for determining problematic collisions.
```

Comment:
all event-values -> eventClass-values renames in the javadocs should be reverted as it's more confusing this way, since you'd have to first find out what event class does. you're more likely to have a rough idea of how stuff relates to event-values, rather than eventClass-values